src/command/FileCommands.cxx

```diff
@@ -234,9 +236,93 @@ handle_read_comments(Client &client, Request args, Response &r)
 	gcc_unreachable();
 }
 
+AllocatedPath
+find_art(AllocatedPath directory)
+{
+    std::vector<std::string> art_names = {
```

Comment:
OK makes sense, ideally we'd allow the user to specify these based on their library configuration too, but I haven't delved into the config files as yet.